notes/05-first-routes.md

```diff
@@ -91,7 +91,9 @@ Paste the HTML below into [`app/templates/login.hbs`](../app/templates/login.hbs
     <div class="w-full max-w-xs">
       <form class="bg-grey-light shadow-md rounded px-8 pt-6 pb-8 mb-4">
         <div class="inline-block relative w-64 mt-2">
+          <label class="sr-only">Select a user</label>
           <select
+            id="select-user"
```

Comment:
Oddly, this is accessible in the starter file but is incorrect in the notes, so is made to be consistent between the two.